src/components/organisms/Home.vue

```diff
@@ -1,17 +1,17 @@
 <template>
-	<div class="home" style="background-color: black;">
+	<div class="home">
 		<homeImageContainer v-bind:imageContainer="imageContainer" v-bind:key="imageContainer.id" v-for="imageContainer in containers"/>
```

Comment:
get key from containers in loop

Reply:
I didn't quite understand what do you mean